observability/opentelemetry-extension/src/main/java/org/wildfly/extension/opentelemetry/extension/OpenTelemetrySubsystemRemove.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * JBoss, Home of Professional Open Source.
+ *
+ * Copyright 2021 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wildfly.extension.opentelemetry.extension;
+
+import org.jboss.as.controller.AbstractRemoveStepHandler;
+import org.jboss.as.controller.OperationContext;
+import org.jboss.as.controller.OperationFailedException;
+import org.jboss.dmr.ModelNode;
+
+/**
+ * Handler responsible for removing the subsystem resource from the model
+ *
+ * @author <a href="jasondlee@redhat.com">Jason Lee</a>
+ */
+public class OpenTelemetrySubsystemRemove extends AbstractRemoveStepHandler {
```

Comment:
This class can be dropped; use ReloadRequiredRemoveStepHandler.

A resource that adds a DUP in its 'add' op can only have it's effect on the runtime removed via a reload or restart, which ReloadRequiredRemoveStepHandler handles.